test/studies/ssca2/rachels/SSCA2_kernels.chpl

```diff
@@ -518,15 +522,15 @@ module SSCA2_kernels
           var tpv = TPV[t];
           var al = tpv.Active_Level;
           coforall loc in Locales do on loc {
-            var level = al[here.id];
-            var prev = level.previous;
+            var level:unmanaged Level_Set? = al[here.id];
```

Comment:
I am OK with this as-is, however I think the following syntax would be nicer:
```chpl
var level = al[here.id]?;
```

Reply:
Right, this would require first solving #13619